src/relay/pass/pattern_util.h

```diff
@@ -379,6 +379,8 @@ Expr MakeStridedSlice(Expr data, Array<Integer> begin, Array<Integer> end, Array
 
 Expr StopFusion(Expr data);
 
+Expr ForceCast(Expr data);
```

Comment:
What does this do?

Reply:
@ZihengJiang

Reply:
Forcibly cast the data to low-precision datatype